macro/macroutils.py

```diff
@@ -57,14 +62,14 @@ def repo_devel_job_data_file(repo_name, distro=None):
     else:
         return os.path.join(doc_path, 'devel_jobs', repo_name, "results.yaml")
 
-def repo_haros_data_file(repo_name, distro=None):
+def repo_report_dir(repo_name, distro=None):
     """
-    Generate filesystem path to haros_report.yaml for repository
+    Generate filesystem path to test results/reports directory for repository
     """
     if distro:
-        return os.path.join(doc_path, distro, 'haros', repo_name, "haros_report.yaml")
+        return os.path.join(doc_path, distro, 'devel_jobs', repo_name, "test_results")
     else:
-        return os.path.join(doc_path, 'haros', repo_name, "haros_report.yaml")
+        return os.path.join(doc_path, 'devel_jobs', repo_name, "test_results")
```

Comment:
Nitpick: instead of repeating almost the same information in both conditional blocks please consolidate them to avoid duplication. E.g.:

```
test_results_dir = doc_path
if distro:
    test_results_dir = os.path.join(test_results_dir, distro)
test_results_dir = os.path.join(test_results_dir, 'devel_jobs', repo_name, 'test_results')
```

Same below in other cases.

Reply:
Thanks.
I changed it for this instance, but I guess reformatting the other cases (which were not affected by this PR) should go in a different PR to avoid confusion.